docs/oss/reading-deepnote-files.md

```
@@ -0,0 +1,805 @@
+```bash
+yq '.project.notebooks[].blocks[] | select(.type == "code" and (.content | contains("pandas"))) | .content' notebook.deepnote
+```
+
+## Editing .deepnote Files
+
+### Safe Edits
+
+These edits are generally safe:
+
+✅ **Change project name:**
+```yaml
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)